Changelog.md

```diff
@@ -1,34 +1,42 @@
-## 4.0.0 - UNRELEASED
+## 4.0.0 - 2024-03-13
 
-## Changed
+### Added
+
+-   Reintroduced triggers functionality.
+-   The configuration in the advance menu to stop saving to disk when disk is
+    running low on disk space.
+
+### Changed
 
 -   Changed export format to `.ppk2`. `.ppk` has been deprecated and importing
     this format will autogenerate a `.ppk2` file next to the `.ppk` file.
--   Exporting of data now works also when the data was loaded from file. This
-    allows partial export to `.csv`.
 -   Removed limitations for recording duration by using the disk to store the
     collected data instead of using RAM.
+-   Performance improvements.
+-   Default path for the session is in the temp folder. This can be changed from
```

Comment:
```suggestion
-   The temporary folder is now the default path for the session. This can be changed from
```

Reply:
he temporary folder is the default path for the session. This can be changed from